### PR TITLE
Add skaffold

### PR DESCRIPTION
**Pull request by rogaldh**

### CODE REVIEW

File: scripts/skaffold.init.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+GCLOUD_SDK_PATH="${HOME}/${GCLOUD_SDK_DIR:-"google-cloud-sdk"}"
+source ${GCLOUD_SDK_PATH}/path.bash.inc
+
+NS=${NAMESPACE:-${NS}}
```
Comment (manifest) [inline scripts/skaffold.init.sh]: Why just don't use `${NAMESPACE}`?

Comment (manifest) [reply scripts/skaffold.init.sh]: We could check the variable contains a value.

File: scripts/skaffold.install.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64
+chmod +x skaffold
+mv skaffold ${HOME}
```
Comment (manifest) [inline scripts/skaffold.install.sh]: Not sure it's good idea to move something to ${HOME}. Not a big problem on travis-ci machine, but kind of unexpected on local one.

File: scripts/skaffold.init.sh (hunk 1 above)
Comment (rogaldh) [reply scripts/skaffold.init.sh]: It was the shorthand been used on desktop: `NS=smth ./skaffold.init.sh`

File: scripts/skaffold.install.sh (hunk 2 above)
Comment (rogaldh) [reply scripts/skaffold.install.sh]: Fixed

**Pull request merged**